src/popup/pages/Assets/AssetDetailsInfo.vue

```diff
@@ -199,19 +215,23 @@ export default defineComponent({
     Tokens,
     IonPage,
     IonContent,
+    LinkButton,
   },
 
   setup() {
     const { sharedAssetDetails } = useAssetDetails();
     const { formatCurrency } = useCurrencies();
 
-    const assetData = computed((): IAsset => sharedAssetDetails.tokenData || {});
+    const assetData = computed(() => sharedAssetDetails.tokenData || {});
```

Comment:
Why the type is removed from the composable?

Reply:
It seems it is not the correct type. We have 9+ TS errors when using the `IAsset` type